apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -204,10 +204,11 @@ public synchronized boolean isInBounds(DateRangeParam range) {
    * call.
    */
   @Scheduled(fixedDelay = 1000, initialDelay = 2000)
-  public synchronized void refreshFilters() {
+  public void refreshFilters() {
     /*
```

Comment:
This the mistake. The design calls for this function to not interfere with requests. By making the call synchronous, it locks the manger and requests have to wait.